Gemfile

```diff
@@ -6,7 +6,7 @@ gem "minitest-autotest", "~> 1.0"
 gem "minitest-focus", "~> 1.1"
 gem "minitest-rg", "~> 5.2"
 gem "autotest-suffix", "~> 1.1"
-gem "rubocop", "<= 0.35.1"
+gem "rubocop", "~> 0.50.0"
```

Comment:
The latest Rubocop is 0.52.1. Is there a reason we're not going all the way?

Reply:
Yes. Rubocop 0.50 is the last version that still supported Ruby 2.0. We have to pin this version so CI will continue to build using Ruby 2.0. I'm waiting to hear if we can drop Ruby 2.0 from CI, but we technically still support it.